Toolchain/serenity.nix

```diff
@@ -32,7 +32,17 @@ stdenv.mkDerivation {
     openssl
     parted
     qemu
-    xlibsWrapper
+
+    freetype
+    fontconfig
+    xorg.xorgproto
+    xorg.libX11
+    xorg.libXt
+    xorg.libXft
+    xorg.libXext
+    xorg.libSM
+    xorg.libICE
```

Comment:
Do we even need the new dependencies? I just removed xlibsWrapper, and Serenity built just fine for me.